Mathlib/Computability/Encoding.lean

```diff
@@ -33,9 +33,13 @@ namespace Computability
 
 /-- An encoding of a type in a certain alphabet, together with a decoding. -/
 structure Encoding (α : Type u) where
+  /-- The alphabet of the encoding. -/
```

Comment:
Small nit: I think doc-strings should end with a period iff they are complete sentences.
This is not one, so should not end with a period. (The same applies to the doc-string of this structure, and the next two fields you're documenting.)